app/layout.tsx

```diff
@@ -16,10 +15,7 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className={inter.className}>
-        <Navbar />
-        {children}
-      </body>
```

Comment:
Why did you removed the Navbar from the Root Layout?

Reply:
I thought it would be better on the page, as there may be pages that don't use the Navbar

Reply:
Could you suggest which pages would not make use of the navbar?

Reply:
Like Error page

Reply:
But with the next 13 dont make to much sense what I said